src/main/java/com/_up/megastore/services/implementations/UserService.java

```diff
@@ -26,19 +26,17 @@ public class UserService implements IUserService {
 
   private final IEmailService emailService;
   private final IUserRepository userRepository;
-  private final ITokenRepository tokenRepository;
   private final PasswordEncoder passwordEncoder;
   private final ITokenService tokenService;
   @Value("${frontend.url}")
   private String frontendURL;
 
   public UserService(IUserRepository userRepository, PasswordEncoder passwordEncoder,
-      IEmailService emailService, ITokenRepository tokenRepository, ITokenService iTokenService) {
+      IEmailService emailService, ITokenService TokenService) {
```

Comment:
Normalmente no te lo corregiría pero ya que vas a hacer otro commit lo hago. Estamos declarando las variables en **camel case**
```suggestion
      IEmailService emailService, ITokenService tokenService) {
```